client/modules/Booking/Booking.js

```diff
@@ -134,9 +135,7 @@ export class Booking extends Component {
       // Placeholder ccv validation.
 //    ccv: this.state.ccv.length > 4,
       startDate: moment(this.startTime).isSameOrAfter(this.state.endDate),
```

Comment:
should be this.state.startDate not this.startTime but thats my original mistake I believe